README.md

```diff
@@ -1,15 +1,86 @@
-# Metadata-Analysis-Workshop
-The documents and code in this repository are for the DPLA Metadata Bootcamp, held on April 18, 2017. More to come...
+# DPLAFest 2017 DPLA Metadata Analysis Workshop
 
-# Workshop Schedule
+Welcome to the open repository, documentation and materials for the DPLAfest 2017 DPLA Metadata Analysis Workshop.
 
-*	Intro/welcome housekeeping etc.: Gretchen (15)
-*	DPLA MAP and the basic requirements/ crosswalk & mapping goals: Gretchen (30)
-*	Command line basics: Audrey (30)
-*	Harvesting OAI feed at the command line: Christina (30) 
-*	break (15)
-*	Metadata analysis: Christina (90 - includes working time)
-*	Other tools to explore: Gretchen (30)
+## About the Workshop
 
-# Requirements:
-(more to come)
+The DPLAfest 2017 DPLA Metadata Analysis Bootcamp for Hubs & Partners will be held **1 PM to 5 PM, Wednesday, April 19, 2017 at the Chicago Public Library’s Harold Washington Library Center** (enter at 401 S. Plymouth Street and read more about space logistics [here](https://dp.la/info/get-involved/dplafest/april-2017/travel/)). Exact room number TBA (we will update this repository and email registered participants).
```

Comment:
Need to confirm address + room number here (presuming it is in the same building as the rest of the DPLAfest).